test/stateDirectivesSpec.js

```diff
@@ -389,6 +389,42 @@ describe('uiStateRef', function() {
       expect(transitionOptions.notify).toBeUndefined();
     }));
   });
+
+  describe('transition expression', function() {
+    beforeEach(inject(function($rootScope, $compile) {
+      el = angular.element('<a ui-sref="contacts.item.detail({id: 5})" ui-sref-on="loading = $transition">Details</a>');
+      scope = $rootScope;
+      scope.loading = null;
```

Comment:
@warrenseymour I thought of a way to fix the tests for the 1.0.x thing until we drop support for it:

```javascript
var _transition = null;
el = angular.element('<a ui-sref="contacts.item.detail({id: 5})" ui-sref-on="load($transition)">Details</a>');
scope.load = function(transition) {
  _transition = transition;
}
```

Then s/`scope.loading`/`_transition`/.